src/components/_input.scss

```diff
@@ -2,33 +2,42 @@
 
 // --- SUI Input --- //
 // @include sui-input;
+// @include sui-input('large');
 
-@mixin sui-input {
+@mixin sui-input($size: 'medium') {
   @include input-placeholder {
     font-style: italic;
```

Comment:
The scope of this PR is to modify inputs and selects in order to fit with button sizes.
i didn't mean to modify input's placeholder, but i guess we can improve it with sass variables as you suggest.